lib/mdx.js

```diff
@@ -24,10 +24,17 @@ const tokenClassNames = {
   comment: 'text-gray-400 italic',
 }
 
+/**
+ * @param {string} type
```

Comment:
Can we exclude jsdoc comments? I want to be consistent across the code and don't want to maintain it. Thanks!